test/defines.hpp

```diff
@@ -1,11 +1,22 @@
 #include <string>
 
 // Correctly formated files
-static const std::string COMPLEX_DBC_FILE 	= "./dbcs/Complex.dbc";
-static const std::string SIMPLE_DBC_FILE	= "./dbcs/Simple.dbc";
+static const std::string COMPLEX_DBC_FILE 	= std::string(TESTDBCFILES_PATH) + "/Complex.dbc";

```

Comment:
So this `TESTDBCFILES_PATH` is now set at compile time with cmake?

Reply:
yes exactly. This makes it more flexible and only at one point the path has to be changed just in case